lib/apivore/swagger_checker.rb

```diff
@@ -53,24 +53,40 @@ def response=(response)
       @response = response
     end
 
+    def tested_mappings
+      result = {}
+      @swagger.each_response do |path, method, response_code, fragment|
+        next unless untested_mappings.dig(path, method, response_code).nil?
+
+        result[path] ||= {}
+        result[path][method] ||= {}
+        result[path][method][response_code] = fragment
+      end
+
+      JSON.parse(JSON.generate(result))
+    end
+
     attr_reader :response, :swagger, :swagger_path, :untested_mappings
 
     private
 
     attr_reader :mappings
 
-    def initialize(swagger_path)
+    def initialize(swagger_path, is_local: false)
       @swagger_path = swagger_path
-      load_swagger_doc!
+      load_swagger_doc!(is_local: is_local)
       validate_swagger!
       setup_mappings!
     end
 
-    def load_swagger_doc!
-      @swagger = Apivore::Swagger.new(fetch_swagger!)
+    def load_swagger_doc!(is_local: false)
```

Comment:
isn't it better to set `@is_local = is_local` and then use it further instead of passing it down?

Reply:
Good idea, done